.github/workflows/scripts/win_build.ps1

```diff
@@ -0,0 +1,74 @@
+# Build script for windows
+
+param (
+    [switch]$clone = $false,
+    [switch]$vulkan = $false,
+    [switch]$develop = $false,
+    [switch]$install = $false,
+    [string]$build = "_build"
+)
+
+$RepoURL = 'https://github.com/taichi-dev/taichi'
+
+function WriteInfo($text) {
+    Write-Host -ForegroundColor Green "[BUILD] $text"
+}
+
+WriteInfo("Install 7Zip")
+Install-Module 7Zip4PowerShell -Force -Verbose -Scope CurrentUser
+
+if ($clone) {
+    WriteInfo("Clone the repository")
+    git clone --recurse-submodules $RepoURL
+    Set-Location .\taichi
+}
+if (-not (Test-Path $build)) {
+    New-Item -ItemType Directory -Path $build
+}
+Push-Location $build
+WriteInfo("Download and extract LLVM")
```

Comment:
Note that the LLVM/clang/Vulkan libs usually are usually downloaded once and placed somewhere outside the repo dir/build dir, (i.e., a more stable place, e.g. `C:\ThirdParty\Libs`). I'd suggest that we add one additional flag to hold all these required dependent libs, WDYT?

In addition, let's make the default behavior not to download these libs?

Reply:
I see, then let's give it a more obvious name, e.g. `libsdir`, `depsdir`, ...

Reply:
the lib location can be specified with `-build` option, and if the library exists, won't download again.

So `-build C:\ThirdParty\Libs` can do the trick.

Reply:
The name is changed to `libdir`

Reply:
Huh, I guess the naming convention here is `$libsDir`? Also, the default value should be empty

Reply:
I've updated the name and default to the current location (`.`)